### PR TITLE
refactor: Standardize Error Schema

### PR DESCRIPTION
**Pull request by tgriesser**
- Closes UNIFY-978

Standardizing the error shape in GraphQL as well as internally in the DataContext / IPC layer

### CODE REVIEW

File: packages/launchpad/src/error/BaseError.vue [hunk 1]
```diff
@@ -1,50 +1,51 @@
...
         data-testid="error-header"
       >
         <slot name="header">
-          {{ headerText }}
+          {{ baseError.title }}
         </slot>
       </h1>
       <!-- eslint-disable vue/multiline-html-element-content-newline  -->
 
       <slot name="message">
```
Comment (lmiller1990) [inline packages/launchpad/src/error/BaseError.vue:17]: Pass in your component with the props here and we should be good

`<OpenInIDE>`

Comment (lmiller1990) [reply packages/launchpad/src/error/BaseError.vue:17]: https://github.com/cypress-io/cypress/pull/19872/commits/dd1ee9af037b56e0aa1d095c0a7a6d5745ad1051

**Comment by jennifer-shehane** [comment]
@tgriesser Can this be closed? Is this outdated by https://github.com/cypress-io/cypress/pull/20323?

**Comment by tgriesser** [comment]
@jennifer-shehane yep, closing in favor of #20323

**Pull request closed without merging**